applications/nrf5340_audio/src/bluetooth/le_audio_cis_headset.c

```diff
@@ -311,7 +211,7 @@ static int lc3_config_cb(struct bt_conn *conn, const struct bt_bap_ep *ep, enum
 			if (dir == BT_AUDIO_DIR_SINK) {
 				LOG_DBG("BT_AUDIO_DIR_SINK");
 				print_codec(codec, dir);
-				le_audio_event_publish(LE_AUDIO_EVT_CONFIG_RECEIVED);
+				le_audio_event_publish(LE_AUDIO_EVT_CONFIG_RECEIVED, conn);
```

Comment:
I see that this event will need to be changed in the bt_stream rewrite regardless, since it does not contain direction. Meaning we wouldn't know if the config was for source or sink. Can add the event for now, since it is only used for debugging